pack/tdc/tdcp.json

```diff
@@ -626,6 +626,27 @@
         "type_code": "event",
         "xp": 3
     },
+    {
+        "code": "11033",
+        "cost": "3",
+        "deck_limit": 2,
+        "faction_code": "seeker",
+        "health": 2,
+        "illustrator": "Alexandre Dainche",
+        "name": "Alton O'Connell",
+        "pack_code": "tdcp",
+        "position": 33,
+        "quantity": 2,
+        "sanity": 1,
+        "skill_intellect": 1,
+        "skill_agility": 1,
```

Comment:
NB `skill_agility` and `skill_intellect` should be switched on most cards in this PR to maintain alphabetical order.